spyder/api/plugins.py

```diff
@@ -119,8 +217,11 @@ def starting_long_process(self, message):
             Message to show in the status bar when the long
             process starts.
         """
-        super(BasePlugin, self)._starting_long_process(message)
+        self.show_status_message(message)
+        QApplication.setOverrideCursor(QCursor(Qt.WaitCursor))
+        QApplication.processEvents()
 
+    # FIXME: Should be end_
```

Comment:
See my comment above.

Reply:
See my comment above

Reply:
Then this needs to be `after_long_process`